scripts/zones/Chocobo_Circuit/npcs/Azainnie.lua

```diff
@@ -0,0 +1,27 @@
+-----------------------------------
+-- Area: Chocobo_Circuit
+-- NPC: Azainnie
+-- !pos -60.623 -14.500 -133.451 70
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    player:startEvent(328)
+end
+
+entity.onEventUpdate = function(player, csid, option)
+end
+
+entity.onEventFinsih = function(player, csid, option)
+if
```

Comment:
this "if" (and others below) is missalligned

Reply:
Yeah, it's unfortunate that CI didn't/doesn't catch this. Available Lua formatters are very very basic, so we're essentially building our own over time. The best thing you can do until our formatted is bulletproof is compare your code to other files in the codebase and see if it matches up, the rest of the Lua code is in fairly good shape these days